translator/reader/read_options.cpp

```diff
@@ -458,7 +476,38 @@ void UsdArnoldReadRenderSettings::Read(const UsdPrim &prim, UsdArnoldReaderConte
                 outputs[j + prevOutputsCount] += std::string(" ") + layerNames[j];
             }
         }
+        // For exr drivers, we need to set the attribute "half_precision"
+        if (!isHalfList.empty() && driverType == "driver_exr") {
```

Comment:
tif should also be able to handle half precision, do we want to add it as well ?

Reply:
not in Arnold. 
Here we're setting the attribute "half_precision" in the arnold driver node type "driver_exr". We don't have such attribute in "driver_tiff".